code/modules/vore/fluffstuff/custom_boxes_vr.dm

```diff
@@ -217,6 +217,19 @@
 		/obj/item/weapon/melee/fluffstuff/wolfgirlsword,
 		/obj/item/weapon/shield/fluff/wolfgirlshield)
 
+// Victor Zisthus: KA-BOOM
+/obj/item/weapon/storage/box/fluff/kaboom
+	name = "KA-BOOM's Stuff"
+	desc = "A box filled with some starting equipment for the android."
+	has_items = list(/obj/item/weapon/spacecash/c500,
+	/obj/item/weapon/spacecash/c200 = 2,
+	/obj/item/weapon/storage/belt/utility/full/fluffsynth,
+	/obj/item/clothing/mask/gas/commando,
+	/obj/item/clothing/mask/balaclava,
+	/obj/item/clothing/accessory/storage/black_vest,
+	/obj/item/weapon/storage/backpack/messenger/black)
```

Comment:
It would probably be better to just give yourself a fluff backpack that has the rest of your stuff in it, and add that to load out as opposed to having a fluff item box. Fluff item boxes are kinda depreciated anyway.

Reply:
Oh my God, that would be much better lmao

I'll change that tonight